.gitignore

```diff
@@ -5,5 +5,6 @@ docker-compose.override.yml
 docker-compose.override.dev.yml
 backend/clients
 frontend/test-results
+backend/data
```

Comment:
would I need any local data to make things happen since this is gitignored? or is this just a place for local data to live once it's generated?

Reply:
actually I'm not sure if this is needed or not, my local had some CSV's in it, which I think were a temp files from qualtrics download